### DISCUSSION TITLE
Installation via Homebrew

### DISCUSSION THREAD

**Discussion by nikolay** [livebud/bud#111] [Ideas]
Many of us try to keep off software installed via `curl | sh`.

**Comment by matthewmueller** [comment]
I'd like to add it to Homebrew soon! Though the recipe itself would still likely just do `curl | sh` under the hood (through homebrew's APIs). Does that work for you or is there a reason you avoid `curl | sh`?

**Reply by nikolay** [reply]
To prevent a supply chain attack. Homebrew uses checksums.

**Reply by matthewmueller** [reply]
Ah, we also use checksums to verify integrity of the downloads: https://github.com/livebud/bud/blob/f0211fc7ab9044c85cc96e040d3f725c3ff7f12e/install.sh#L44

**Reply by nikolay** [reply]
@matthewmueller You can't have the checksum in the script itself - what you've done is only a consistency check, not a security check.

**Reply by matthewmueller** [reply]
@nikolay thanks for the info! Could you share a bit more details about what you mean by security check?
